cms/templates/certificate_page.html

```diff
@@ -44,7 +44,7 @@ <h2>Congratulations, {{ learner_name }}!</h2>
           <div class="certificate">
             <div class="certificate-head">
               <div class="certificate-b-logo">
-                <img src="{% static 'images/certificates/certificate-bootcamps-logo.png' %}" alt="Certificate Bootcamp Logo">
```

Comment:
When we add the secondary or partner logo in the certificate, the design becomes a little out. e.g. Check https://bootcamp-rc.odl.mit.edu/certificate/ff9a1737-c9c1-4200-b684-55382a586ac7/

And how it looks with this PR:

<img width="983" alt="image" src="https://github.com/mitodl/bootcamp-ecommerce/assets/34372316/97b60a74-016d-4e0b-b718-d172bbc1ea4e">


Notice the spacing below the bootcamps logo

Reply:
Updated in https://github.com/mitodl/bootcamp-ecommerce/pull/1484/commits/0504a2bf7756fff2b18460a0245694069f4c3cc2

![Opera Snapshot_2024-02-13_154914_bc odl local](https://github.com/mitodl/bootcamp-ecommerce/assets/52656433/ffe02f43-115a-44c9-83b1-7a3302e2a121)
![Opera Snapshot_2024-02-13_154954_bc odl local](https://github.com/mitodl/bootcamp-ecommerce/assets/52656433/7b41b328-9587-46c2-8019-8ee56283cb65)